scalableminds/webknossos
# Drag'n'drop import of multiple NML files

### URL of deployed dev instance (used for testing):
- https://dndnml.webknossos.xyz

### Steps to test:
- upload one or several NML files via:
  - drag and drop into the tracing view
  - the usual "import nml" button in the trees tab (in the modal which opens via this button, you can either use dnd or you can click the area to open a file picker)
- in the import modal, you can check the option to auto-create groups for each file. try it with and without

### Issues:
- fixes #2901 
- fixes #2891 

------
